README.md

```diff
@@ -117,23 +116,23 @@ results = (
 ```
 <img src="towhee_example.png" style="width: 60%; height: 60%">
 
-Learn more examples from [Towhee bootcamp](https://codelabs.towhee.io/)
+Learn more examples from the [Towhee Bootcamp](https://codelabs.towhee.io/).
 
 ## Core Concepts
 
 Towhee is composed of four main building blocks - `Operators`, `Pipelines`, `DataCollection API` and `Engine`.
 
-- __Operator__: An operator is a single building block of neural data processing pipelines. Different implementations of operators are categorized by tasks, with standard task interface. An operator can be a deep learning model, a data processing method, or a Python function.
+- __Operators__: An operator is a single building block of neural data processing pipelines. Different implementations of operators are categorized by tasks, with a standard task interface. An operator can be a deep learning model, a data processing method, or a Python function.
```

Comment:
- __Operators__: An operator is a single building block of a neural data processing pipeline. Different implementations of operators are categorized by tasks, with each task having a standard interface. An operator can be a deep learning model, a data processing method, or a Python function.